src/udev/udev-builtin-blkid.c

```diff
@@ -58,6 +58,10 @@ static void print_property(sd_device *dev, bool test, const char *name, const ch
                 blkid_encode_string(value, s, sizeof(s));
                 udev_builtin_add_property(dev, test, "ID_FS_LABEL_ENC", s);
 
+        } else if (startswith(name, "FS")) {
+                strscpyl(s, sizeof(s), "ID_FS_", name, NULL);
```

Comment:
Maybe, it is better to drop `FS` prefix: e.g. `ID_FS_FSSIZE` -> `ID_FS_SIZE`.

Reply:
Signed-off-by tag dropped. Changed to `name + 2`. Initially I thought about it but left it, to exactly correspond to libblkid names, but that looks better.